lib/util/send-request.js

```diff
@@ -45,7 +45,8 @@ export default function sendRequest( params, query, body, fn ) {
 	if ( query.apiVersion ) {
 		params.apiVersion = query.apiVersion;
 		debug( 'apiVersion: %o', params.apiVersion );
-		delete query.apiVersion;
+		// copying object here so we are not mutating original arguments
+		query = Object.assign( {}, query, { apiVersion: undefined } );
```

Comment:
You can [use the spread operator](http://redux.js.org/docs/recipes/UsingObjectSpreadOperator.html) for this. `query = { ...query, { apiVersion: undefined } };`